# Automatically Generated in 2017-06-12 22:33:47.453237
# MESSAGE: CONFIG_PARAM_SET_FLOAT
# Description:Set new config parameter float value
uint8 ID = 202
uint8 sysid
uint8 compid
uint8 config_id
uint8 param_id
float32 value
